data/data-pipeline/data_pipeline/etl/score/constants.py

```diff
@@ -60,40 +63,53 @@
 
 # Column subsets
 CENSUS_COUNTIES_COLUMNS = ["USPS", "GEOID", "NAME"]
-TILES_SCORE_COLUMNS = [
-    field_names.GEOID_TRACT_FIELD,
-    field_names.STATE_FIELD,
-    field_names.COUNTY_FIELD,
-    field_names.TOTAL_POP_FIELD,
-    field_names.SCORE_D + field_names.PERCENTILE_FIELD_SUFFIX,
-    field_names.SCORE_D + field_names.TOP_25_PERCENTILE_SUFFIX,
-    field_names.SCORE_E + field_names.PERCENTILE_FIELD_SUFFIX,
-    field_names.SCORE_E + field_names.TOP_25_PERCENTILE_SUFFIX,
-    field_names.SCORE_G_COMMUNITIES,
-    field_names.SCORE_G,
-    field_names.SCORE_L_COMMUNITIES,
-    field_names.SCORE_L + field_names.PERCENTILE_FIELD_SUFFIX,
-    field_names.POVERTY_FIELD + field_names.PERCENTILE_FIELD_SUFFIX,
-    field_names.HIGH_SCHOOL_ED_FIELD + field_names.PERCENTILE_FIELD_SUFFIX,
-    field_names.LINGUISTIC_ISO_FIELD + field_names.PERCENTILE_FIELD_SUFFIX,
-    field_names.UNEMPLOYMENT_FIELD + field_names.PERCENTILE_FIELD_SUFFIX,
-    field_names.HOUSING_BURDEN_FIELD + field_names.PERCENTILE_FIELD_SUFFIX,
-    field_names.DIABETES_FIELD + field_names.PERCENTILE_FIELD_SUFFIX,
-    field_names.ASTHMA_FIELD + field_names.PERCENTILE_FIELD_SUFFIX,
-    field_names.HEART_DISEASE_FIELD + field_names.PERCENTILE_FIELD_SUFFIX,
-    field_names.LOW_LIFE_EXPECTANCY_FIELD + field_names.PERCENTILE_FIELD_SUFFIX,
-    field_names.TRAFFIC_FIELD + field_names.PERCENTILE_FIELD_SUFFIX,
-    field_names.FEMA_RISK_FIELD + field_names.PERCENTILE_FIELD_SUFFIX,
-    field_names.ENERGY_BURDEN_FIELD + field_names.PERCENTILE_FIELD_SUFFIX,
-    field_names.WASTEWATER_FIELD + field_names.PERCENTILE_FIELD_SUFFIX,
-    field_names.LEAD_PAINT_FIELD + field_names.PERCENTILE_FIELD_SUFFIX,
-    field_names.DIESEL_FIELD + field_names.PERCENTILE_FIELD_SUFFIX,
-    field_names.PM25_FIELD + field_names.PERCENTILE_FIELD_SUFFIX,
+
+TILES_ROUND_NUM_DECIMALS = 2
+# Tiles data: full field name, tile index name
+TILES_SCORE_COLUMNS = {
```

Comment:
@esfoobar-usds -- where did you get these fields from? These are unfortunately not the right fields. Please see "Indicators for UI and Export" spreadsheet.

Reply:
Oh, I guess you hadn't attempted to get these right yet: 

> I have completed the logic piece of this. What's now left to do is double check and add/substract the fields from the source of truth excel.